keps/sig-network/2009-dns-autopath/kep.yaml

```diff
@@ -0,0 +1,24 @@
+---
+title: DNS Autopath in PodSpec
+kep-number: 2009 
+authors:
+  - "@prameshj"
+owning-sig: sig-network
+participating-sigs:
+  - sig-network
+status: implementable
+creation-date: 2019-04-15
+last-updated: 2020-09-23
+reviewers:
+  - "@thockin"
+  - "@bowei"
+  - "@johnbelamaric"
+approvers:
+  - "@thockin"
+  - "@bowei"
+  - "@johnbelamaric"
+stage: alpha
+latest-milestone: "v1.21"
+milestone:
+  alpha: "v1.21"
```

Comment:
Change this back to `1.20` and let's change this after the 6th of October if this KEP doesnt get into this release.